docker-compose.yml

```diff
@@ -1,20 +1,22 @@
 version: "3"
 services:
-  editor:
-    build:
-      context: .
-      args:
-        USE_FIXTURES: 'false'
-        TRELLIS_BASE_URL: http://localhost:8080
-    environment:
-      INDEX_URL: http://search:9200
-    ports:
-      - 8000:8000
-    depends_on:
-      - platform
-      - pipeline
+  # editor:
```

Comment:
Why is the editor image commented out? This is needed when deploying on AWS.

Reply:
How is docker-compose being used for AWS deployment?

Reply:
> How is docker-compose being used for AWS deployment?

Likely I'm remembering an earlier iteration; this would break testing locally with the editor image forwarding traffic for search to the local containers.